packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessabilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,110 @@
+
+Icons that perform some operation, like an icon that is part of a hyperlink or a button, are said to be functional icons. Functional icons, like informative icons, **must have alternative text in code** to ensure that assistive technologies pass the icon's meaning to the user.
+
+<FunctionalIconExample/>
+
+In the example above, links to view a folder in three different ways are given: as a list (<IconList/>), as tiles (<IconTile/>), and as a tree map (<IconTreemap/>). With the title provided to each icon, the user will hear:
+
+<p><em>"View folder as, link, List view, link, Tile view, link, Treemap view"</em></p>
+
+#### Groups of Icons
+
+Sometimes icons only have meaning in the context of other icons. Here is one example where multiple icons are used to create a single five-star rating system:
+
+<GroupOfIconsExample/>
+
+One technique to make groups of images accessible is shown above - to give the first image in the group a meaning but mark the other images as presentational (so that they will be ignored by screen readers). This technique results in the user hearing:
+
+_"Rating: Three and a half stars out of five"_
+
+#### Icons Containing Text
+
+Terra does not provide Icons containig text, but you might encounter such a scenario. One example found commonly is a company logo that includes the company's name in the logo graphic. For more information see https://www.w3.org/WAI/tutorials/images/textual/.
+
+### Decorative Use of Icons
+
+In decorative use, an icon provides no additional information or context. Meaning that, **without the decorative icon, the user would still know the meaning and purpose of content**.
+
+<DecorativeIconExample/>
+
+The factory icon (<IconManufacturer/>) above is purely decorative. It provides an extra visual cue but is not necessary for the user to understand that the text, _2800 Rock Creek Pkwy…_, is the address of the manufacturer.
+
+Since the icon is decorative, its code must set `title=""` to ensure screen readers do not announce the icon's presence. If the title is left out, then the user will hear "Image Manufactured at …". If the title is set to `title="Manufactured at"` like in the Informative example above, then the user would hear "Manufactured at Manufactured at…". Neither of these are correct. The correct choice is to set `title=""` so that the user hears:
+
+<p><em>"Manufactured at 2800 Rock Creek Pkwy Kansas City, MO 64117"</em></p>
+
+### Code Considerations
+
+- Appropriately code each icon as being meaningful or decorative. Ensure each meaningful icon has alternative text provided by the content creator.
+- Do not overwrite theme colors unless UX provides custom colors. All custom colors must pass WCAG 2.1 color accessibility requirements.
+
+<Whitespace />
+
+### Content Considerations
+
+- Specify **every** icon instance as meaningful or decorative. Provide developers with clear and concise alternative text for meaningful icons that accurately reflect the icon's meaning or purpose.
```

Comment:
To clearly distinguish between the bolded main points in this document, "every" could be in italics instead,

Reply:
Will change.